client/components/public/Gear.jsx

```diff
@@ -0,0 +1,223 @@
+import { Meteor } from 'meteor/meteor';
+import React, { Component } from 'react';
+
+import Slider from 'react-slick';
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css"
+
+import { Container, Segment, Header, Icon, Modal, Grid } from 'semantic-ui-react';
+import LinkButton from '../imports/LinkButton';
+
+const { eventYear } = Meteor.settings.public;
+
+const link = `https://commerce.cashnet.com/TheGreatPuzzleHunt${eventYear}`;
+
+const buyButton = (  
+  <LinkButton as='a' href={link}
+    size="huge" color="orange" target="_blank"
+    icon={<Icon name="shopping cart" />}
+    content="Buy"
+  />
+);
+
+const xxlStr = ", +$2 for 2XL or larger.";
+
+const forward = {
+  "ctm": 0,
+  "ctw": 1,
+  "cty": 1,
+  "btm": 0,
+  "btw": 0,
+  "lstw": 0,
+  "lstm": 1,
+  "fcu": 0,
+  "hu": 1,
+  "hy": 1,
+  "qzu": 1
+}
+
+const titles = {
+  "ctm": "Men's Cotton Tee",
+  "ctw": "Women's Cotton Tee",
+  "cty": "Youth Cotton Tee",
+  "btm": "Men's Cotton/Poly Blend Tee",
+  "btw": "Women's Cotton/Poly Blend Tee",
+  "lstw": "Women's Long Sleeve Cotton Tee",
+  "lstm": "Men's Long Sleeve Cotton Tee",
+  "fcu": "Unisex Crew Sweatshirt",
+  "hu": "Unisex Hoodie Sweatshirt", 
+  "hy": "Youth Hoodie",
+  "qzu": "Unisex Quarter-zip Collar Sweatshirt",
+}
+
+const materials = {
+  "ctm": "100% Cotton",
+  "ctw": "100% Cotton",
+  "cty": "100% Cotton",
+  "btm": "50/50 Cotton/Polyester",
+  "btw": "50/50 Cotton/Polyester",
+  "lstw": "100% Cotton",
+  "lstm": "100% Cotton",
+  "fcu": "50/50 Cotton/Polyester",
+  "hu": "50/50 Cotton/Polyester",
+  "hy": "50/50 Cotton/Polyester",
+  "qzu": "50/50 Cotton/Polyester"
+};
+
+const prices = {
+  "ctm": "$14" + xxlStr,
+  "ctw": "$14" + xxlStr,
+  "cty": "$14" + xxlStr,
+  "btm": "$15" + xxlStr,
+  "btw": "$15" + xxlStr,
+  "lstw": "$17" + xxlStr,
+  "lstm": "$17" + xxlStr,
+  "fcu": "$20" + xxlStr,
+  "hu": "$25" + xxlStr,
+  "hy": "$25" + xxlStr,
+  "qzu": "$30" + xxlStr
+}
+
+const sizes = {
+  "ctm": "S-5XL",
+  "ctw": "XS-3XL",
+  "cty": "XS-L",
+  "btm": "XS-4XL",
+  "btw": "XS-4XL",
+  "lstw": "S-3XL",
+  "lstm": "S-5XL",
+  "fcu": "S-5XL",
+  "hu": "S-5XL",
+  "hy": "S-XL",
+  "qzu": "S-3XL"
+}
+
+Gear = class Gear extends Component {
```

Comment:
Do you need the `Gear =` here? I think `class X extends Y` is sufficient

Reply:
You actually get a "Gear not defined" error from the routes if you don't include that! Tripped me up for a while.